src/inverted-index.js

```diff
@@ -1,85 +1,126 @@
-    return this.indexObject;
+   /**
+    * getIndex
+    *
+    * Returns the index of a JSON file
+    *
+    * @params {string} filename
+    * @returns {object}
+    */
+   getIndex(filename) {
+     return this.files[filename];
    }
 
-  /**
-   * Accepts a word(s), searches the word(s) in the file
-   * specified and returns it's index position.
-   * @params filename, words
-   * @returns {}
-   */
-  searchIndex(filename, ...words) {
-    const indexObject = this.indexObject;
-    const terms = words.toString().split(',');
-    const searchTerms = {};
-    const results = {};
-    if (terms.length > 1) {
-      terms.forEach((term) => {
-        if (term in indexObject) {
-          results[term] = indexObject[term];
-        } else {
-          results[term] = ['Not Found', 'Not Found'];
-        }
-      });
-      searchTerms[filename] = results;
-    } else {
-      if (terms in indexObject) {
-        results[terms] = indexObject[terms];
-      }
-      searchTerms[filename] = results;
-    }
-    return searchTerms;
-  }
+   /**
+    * searchIndex
+    *
+    * Search specified words in the specified index and
+    * returns the index of the searched words.
+    *
+    * @param {string}
+    * @param {array}
+    * @returns {object}
+    */
+   searchIndex(filename, ...words) {
+     const indexObject = this.files[filename];
+     const terms = words.toString().split(',');
+     const searchTerms = {};
+     const results = {};
+     if (terms.length > 1) {
+       terms.forEach((term) => {
+         if (term in indexObject) {
+           results[term] = indexObject[term];
+           searchTerms[filename] = results;
+         } else {
+           results[term] = ['Not Found', 'Not Found'];
+           searchTerms[filename] = results;
+         }
+       });
+     } else if (terms in indexObject) {
+       results[terms] = indexObject[terms];
+       searchTerms[filename] = results;
+     } else {
+       results[terms] = ['Not Found', 'Not Found'];
+       searchTerms[filename] = results;
+     }
+     return searchTerms;
+   }
 
-  isJson() {
-    try {
-      const stringFile = JSON.stringify(this.content);
-      JSON.parse(stringFile);
-    } catch (e) {
-      return false;
-    }
-    return true;
-  }
-}
+   /**
+    * isJson
+    *
+    * Checks if a file is a valid JSON file and returns
+    * true if the file is valid and false if it is not.
+    *
+    * @param {object}
+    * @returns {bool}
+    */
+   isJson(file) {
+     let result;
+     try {
+       const stringFile = JSON.stringify(file);
+       const isArray = JSON.parse(stringFile);
+       isArray.some((fileObject) => {
+         if (fileObject.title && fileObject.text) {
+           result = true;
+         } else {
+           result = false;
+           return true;
+         }
+       });
+       return result;
+     } catch (err) {
+       return false;
+     }
+   }
+ }
 
-exports.InvertedIndex = InvertedIndex;
+ exports.InvertedIndex = InvertedIndex;
```

Comment:
### Critical

@andela-wmaina Use the es6 specified approach for exporting classes. i.e `export class` and leverage Babel if you need to.